contrib/sssd.spec.in

```diff
@@ -1131,16 +1135,18 @@ getent passwd sssd >/dev/null || useradd -r -g sssd -d / -s /sbin/nologin -c "Us
 # systemd
 %post common
 %systemd_post sssd.service
+%systemd_post sssd-update-confdb.socket
 %systemd_post sssd-secrets.socket
 
 %preun common
 %systemd_preun sssd.service
+%systemd_preun sssd-update-confdb.socket
 %systemd_preun sssd-secrets.socket
 
 %postun common
 %systemd_postun_with_restart sssd.service
+%systemd_postun_with_restart sssd-update-confdb.socket
 %systemd_postun_with_restart sssd-secrets.socket
-%systemd_postun_with_restart sssd-secrets.service
```

Comment:
systemd_postun_with_restart  will try to restart sssd-secrets in case of upgrade.
`systemctl try-restart` does nothing if units are not running.

We should not rely on re-starting the service on demand when needed.
and other projects do the same.

e.g. cockpit has following scripts for cocpit-ws

```
%postun ws
%systemd_postun_with_restart cockpit.socket
%systemd_postun_with_restart cockpit.service
```